ouroboros-consensus/src/Ouroboros/Storage/VolatileDB/Util.hs

```diff
@@ -74,20 +75,32 @@ wrapFsError :: Monad m
 wrapFsError fsErr volDBErr action =
     tryVolDB fsErr volDBErr action >>= either (throwError volDBErr) return
 
--- Throws an error if one of the given file names does not parse.
+-- | Parses the 'FsPath' of each filename and zips them together.
+-- If any 'FsPath' does not parse, we return an error.
+parseAllFds :: forall blockId.
+               [FsPath]
+            -> Either (VolatileDBError blockId) [(FileId, FsPath)]
+parseAllFds = mapM go
+  where
+    go :: FsPath -> Either (VolatileDBError blockId) (FileId, FsPath)
```

Comment:
Rename `go` to `parseFd`. Two reasons: (1) `go` is mostly used for functions that do explicit recursion, which is not the case here, the recursion is handled by `mapM`; (2) we mostly use `go` when we can't think of a good name, but in this case there is a  good candidate :slightly_smiling_face: